ui/src/containers/SampleListViewContainer.jsx

```diff
@@ -430,27 +374,29 @@ class SampleListViewContainer extends React.Component {
   /**
    *  Clears the filter
    */
-  sampleGridClearFilter() {
-    this.props.filter({
-      inQueue: false,
-      notInQueue: false,
-      collected: false,
-      notCollected: false,
-      limsSamples: false,
-      filterText: '',
-      cellFilter: '',
-      puckFilter: '',
-    });
+  function sampleGridClearFilter() {
+    dispatch(
+      filterAction({
+        inQueue: false,
+        notInQueue: false,
+        collected: false,
+        notCollected: false,
+        limsSamples: false,
+        text: '',
```

Comment:
The change from `filterText` to `text` is still present here.

Reply:
> The change from `filterText` to `text` is still present here.
 
Yes, filterText to text was legit, as it was define as text in the Store
`filterText: { text: ReactDOM.findDOMNode(this.filterInput).value.trim() },`
i think that why they had to do the mapping and use ReactDOM.findDOMNode.

now i think you notice  filterText is a better key name, we can always change it in the Store. 
not in the component as it was.

Reply:
Oh I get you, thanks for clarifying :+1: